### PR TITLE
Newly Added Feeds 

### PR DESCRIPTION
**Pull request by nabihanaqvie**

### CODE REVIEW

File: fixtures/feedly.opml [hunk 1]
```diff
@@ -39,5 +39,32 @@
 		<outline type="rss" text="BBC Turkish - Haberler, Son Dakika Haberleri, Haber-Analiz, Video" title="BBC Turkish - Haberler, Son Dakika Haberleri, Haber-Analiz, Video" xmlUrl="http://feeds.bbci.co.uk/turkce/rss.xml" htmlUrl="http://www.bbcturkce.com"/>
 		<outline type="rss" text="BBC Cymru Fyw - Newyddion a mwy" title="BBC Cymru Fyw - Newyddion a mwy" xmlUrl="http://feeds.bbci.co.uk/cymrufyw/rss.xml" htmlUrl="http://www.bbc.co.uk/cymrufyw"/>
 		<outline type="rss" text="BBC Chinese - 主页" title="BBC Chinese - 主页" xmlUrl="http://feeds.bbci.co.uk/zhongwen/simp/rss.xml" htmlUrl="http://www.bbcchinese.com"/>
+		<outline type="rss" text="presstv.ir" title = "presstv.ir" xmlUrl= "https://www.presstv.ir/RSS" htmlUrl = "https://www.presstv.ir/">
```
Comment (pdeziel) [inline fixtures/feedly.opml]:
```suggestion
		<outline type="rss" text="presstv.ir" title="presstv.ir" xmlUrl="https://www.presstv.ir/RSS" htmlUrl="https://www.presstv.ir/">
```

**Review by pdeziel — COMMENTED** [review]
Hi @nabihanaqvie, thanks for contributing to Baleen! It will be great to have all these new feeds to scrape from.

When running the code from your branch I did get an error, it appears that the opml parsing failed on the new version of `feedly.opml`. This should be an easy fix; we just need to make sure that the opml formatting is correct. I've made one suggestion below as a guideline, but what I would recommend is to copy the file into an opml formatter (e.g., https://beautifytools.com/opml-beautifier.php) so you can see exactly where all the errors are. Also there's at least one spot where we have `_htmlUrl` as an attribute instead of `htmlUrl` which will need to be corrected.

For `feedly.json`, JSON parsing is more forgiving of whitespace so it will probably work, but it would be really nice to have consistent formatting throughout the file. Once quick way to fix that would be to throw it into a JSON formatter like https://jsonformatter.curiousconcept.com/#.

File: fixtures/feedly.opml [hunk 2]
```diff
@@ -39,5 +39,32 @@
...
+		<outline type="rss" text="tehrantimes.com" title="tehrantimes.com" xmlUrl="https://www.tehrantimes.com/rss" htmlUrl="https://www.tehrantimes.com/"/>
+		<outline type="rss" text="iribnews.ir" title="iribnews.ir" xmlUrl="https://www.iribnews.ir/fa/rss" htmlUrl="https://iribnews.ir/"/>
+		<outline type="rss" text="jang.com.pk" title="jang.com.pk" xmlUrl="https://jang.com.pk/rss" htmlUrl="https://jang.com.pk/"/>
+		<outline type="rss" text="aajtak.in" title="aajtak.in" xmlUrl="https://www.aajtak.in/rssfeeds/?id=home" htmlUrl="https://www.aajtak.in/"/>
+		<outline type="rss" text="sundaytimes.lk" title="sundaytimes.lk" xmlUrl="https://www.aajtak.in/rssfeeds/?id=home" htmlUrl="https://www.sundaytimes.lk/"/>
+		<outline type="rss" text="aljazeera.com" title="aljazeera.com" xmlUrl="https://www.aljazeera.com/xml/rss/all.xml"  htmlUrl="https://www.aljazeera.com/"/>
+		<outline type="rss" text="almanar.com.lb" title="almanar.com.lb" xmlUrl="https://www.almanar.com.lb/rss" htmlUrl="https://www.almanar.com.lb/"/>
+		<outline type="rss" text="english.pnn.ps" title="english.pnn.ps" xmlUrl ="http://english.pnn.ps/feed/" htmlUrl="http://english.pnn.ps/"/>
```
Comment (pdeziel) [inline fixtures/feedly.opml]:
```suggestion
		<outline type="rss" text="english.pnn.ps" title="english.pnn.ps" xmlUrl="http://english.pnn.ps/feed/" htmlUrl="http://english.pnn.ps/"/>
```

File: fixtures/feedly.opml [hunk 3]
```diff
@@ -39,5 +39,32 @@
...
+		<outline type="rss" text="mainichi.jp/english/" title="mainichi.jp/english/" xmlUrl="https://mainichi.jp/english/rss/" htmlUrl="https://mainichi.jp/english/"/>
+		<outline type="rss" text="staitstimes.com/global" title="staitstimes.com/global" xmlUrl="https://www.straitstimes.com/RSS-Feeds" htmlUrl="https://www.straitstimes.com/global"/>
+		<outline type="rss" text="taipeitimes.com" title="taipeitimes.com" xmlUrl="https://www.taipeitimes.com/xml/index.rss" htmlUrl="http://www.taipeitimes.com/"/>
+		<outline type="rss" text="manilastandard.net" title="manilastandard.net" xmlUrl="https://manilastandard.net/rss" htmlUrl="https://manilastandard.net/"/>
+		<outline type="rss" text="malaymail.com" title="malaymail.com" xmlUrl="https://www.malaymail.com/feed" htmlUrl="https://www.malaymail.com/"/>
+		<outline type="rss" text="vietnamnews.vn" title="vietnamnews.vn" xmlUrl="https://vietnamnews.vn/rss" htmlUrl="https://vietnamnews.vn/"/>
+		<outline type="rss" text="themoscowtimes.com" title="themoscowtimes.com" xmlUrl="https://www.themoscowtimes.com/page/rss" htmlUrl ="https://www.themoscowtimes.com/"/>
```
Comment (pdeziel) [inline fixtures/feedly.opml]:
```suggestion
		<outline type="rss" text="themoscowtimes.com" title="themoscowtimes.com" xmlUrl="https://www.themoscowtimes.com/page/rss" htmlUrl="https://www.themoscowtimes.com/"/>
```

**Review by pdeziel — CHANGES_REQUESTED** [review]
Thanks for making the changes! I have two more revisions below but after that we should be good to merge!

File: fixtures/feedly.opml (hunk 3 above)
Comment (nabihanaqvie) [reply fixtures/feedly.opml]: Apologies for the inconvenience and thank you for your patience.

**Review by pdeziel — APPROVED** [review]
Thanks again for contributing @nabihanaqvie, I'll go ahead and merge!

**Pull request merged**